maqs-selenium/src/test/java/com/cognizantsoftvision/maqs/selenium/SeleniumDriverManagerUnitTest.java

```diff
@@ -14,7 +13,7 @@
 /**
  * The type Selenium driver manager test.
  */
-public class SeleniumDriverManagerUnitTest extends BaseGenericTest {
+public class SeleniumDriverManagerUnitTest extends BaseSeleniumTest {
```

Comment:
Why add a specific base when generic is all we need?

Reply:
reverted back to baseGeneric